### PR TITLE
added the fix for srcset in img tag

### PR DESCRIPTION
**Pull request by gkumar9891**
added a fix for srcset in img tag

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 var assert = require("assert");
 var absolution = require('../index.js');
+const exp = require("constants");
```
Comment (BoDonkey) [inline test/test.js:3]: Why is this being imported?

File: test/test.js [hunk 2]
```diff
@@ -94,4 +95,41 @@ describe('absolution', function() {
     });
     assert.equal(result, expected);
   });
+
+  it('should handle the scrset in img', function() {
+    const result = absolution(`<img src="cat.jpg" alt="cat"
```
Comment (BoDonkey) [inline test/test.js:100]: I would like to see this test match the test in the issue. It is typically a good pattern to demonstrate that the original problem raised in the issue has been fixed by your code change.

**Review by BoDonkey — COMMENTED** [review]
Thanks for doing this!
I would like to see at least one more test (or maybe one for `` and one for `<picture>`) that shows that if you have a properly configured srcset that it isn't mangled. So if you pass the expected from the issue in as the dirty HTML that it comes out as identical to the input.

File: test/test.js (hunk 1 above)
Comment (gkumar9891) [reply test/test.js:3]: I will remove it, my vs code automatically imported that.

File: test/test.js (hunk 2 above)
Comment (gkumar9891) [reply test/test.js:100]: I didn't understand what I need to do?

**Review by BoDonkey — APPROVED** [review]
Looks good to me now. Thanks!

File: index.js [hunk 3]
```diff
@@ -20,7 +20,21 @@ var absolution = module.exports = function(input, base, options) {
     onopentag: function(name, attribs) {
       _.forEach(options.urlAttributes, function(attr) {
         if (_.has(attribs, attr) && attribs[attr].trim()) {
-          attribs[attr] = url.resolve(base, attribs[attr]);
+          if(attr === 'srcset') {
```
Comment (boutell) [inline index.js:23]: Space before (

**Review by boutell — APPROVED** [review]
Thanks!

File: index.js (hunk 3 above)
Comment (gkumar9891) [reply index.js]: done.

**Pull request merged**